uwblueprint/jump-math
# Boya/Admin_User_Table_Component

## Notion ticket link
https://www.notion.so/uwblueprintexecs/47bf4727a2444bccad4a91952898b8f3?v=9658436bf9be409e90581631a7155f16&p=f9a92055e30a42518c8e645573942708&pm=s



## Implementation description
* created the admin user table component



## Steps to test
1. go to any empty page, add the AdminUserTable component in and the following array to test: 

const adminUserArray = [
    {
      firstName: "Albert",
      lastName: "Kuhl",
      email: "albertkuhl@jumpmath.ca",
    },
    {
      firstName: "Albert",
      lastName: "Kuhl",
      email: "albertkuhl@jumpmath.ca",
    },
    {
      firstName: "Albert",
      lastName: "Kuhl",
      email: "albertkuhl@jumpmath.ca",
    },
    {
      firstName: "Albert",
      lastName: "Kuhl",
      email:
        "The quick brown fox jumps over the lazy dog is an English-language pangram—a sentence that contains all of the letters of the English alphabet. Owing to its existence, Chakra was created.",
    },
  ];
  
Verify the following table is displayed 
<img width="1237" alt="Screen Shot 2022-10-19 at 7 19 03 PM" src="https://user-images.githubusercontent.com/82241706/196822491-4baeb68d-2552-414b-91e2-f01d72fd96e7.png">




## What should reviewers focus on?
* if the implementation matches the design (check the onHover behavior) 
* https://www.figma.com/file/cRjJxm8087kkymfH4ieKau/Platform-Working-File?node-id=277%3A8172


## Checklist
